### ISSUE TITLE
Comptability with Twig 3.9

### ISSUE THREAD

**Issue by majermi4** [giorgiopogliani/twig-components#22]
Hi, I've noticed that our app started failing after third-party libraries upgrade and I traced it to Twig library upgrade from 3.8 to 3.9. 

I suspect it's something related to the twig components lexer?

**Comment by giorgiopogliani** [comment]
yeah, ours too I just pushed an update to limit twig dependency below 3.9

They changed the behaviour of nodes to use `yield` instead of `echo` for some reason I don't know yet.

**Comment by stuible** [comment]
Hey, thanks for the incredible project, I've been a big fan of using it with Craft CMS!  However, the latest version of Craft requires Twig 3.14.0 which is incompatible with Twig Components.  Is there a timeline for releasing a version that is compatible?  Anything I can help with?

**Comment by giorgiopogliani** [comment]
Hey! It seems that Twig had a bug at one point, and the absence of the `YieldReady` attribute wasn’t working properly and so I created the PR. Now, I’m using it with Craft 5 and Twig 3.14, and it seems to be working fine, I'm using `dev-master`, I am not encountering issues so I probably tag a new version.

Regarding the `YieldReady` situation, I’m not sure if it will become mandatory at some point, but when I tried to rewrite the `echo` statements with `yield` I encountered some problems, so at the moment it’s still a work in progress.
